From daa7d1b314dd159ecc1a3ca4746d6ea9bb4a7adb Mon Sep 17 00:00:00 2001
From: Rex Lu <rex.lu@mediatek.com>
Date: Wed, 29 Nov 2023 13:56:52 +0800
Subject: [PATCH 023/146] mtk: mt76: mt7996: Remove wed_stop during L1 SER

Align logan L1 SER flow. During L1 SER, didn't need to close wed interrupt.

Signed-off-by: Rex Lu <rex.lu@mediatek.com>
---
 mt7996/mac.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/mt7996/mac.c b/mt7996/mac.c
index 0c1fdd5b..3303d32b 100644
--- a/mt7996/mac.c
+++ b/mt7996/mac.c
@@ -1877,12 +1877,6 @@ void mt7996_mac_reset_work(struct work_struct *work)
 	dev_info(dev->mt76.dev,"\n%s L1 SER recovery start.",
 		 wiphy_name(dev->mt76.hw->wiphy));
 
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed_hif2))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed_hif2);
-
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed);
-
 	ieee80211_stop_queues(mt76_hw(dev));
 	if (phy2)
 		ieee80211_stop_queues(phy2->mt76->hw);
-- 
2.45.2

